src/main/java/org/spongepowered/common/mixin/core/server/MixinMinecraftServer.java

```diff
@@ -571,40 +571,6 @@ public void onServerTickStart(CallbackInfo ci) {
 
     @Inject(method = "tick", at = @At(value = "RETURN"))
     public void onServerTickEnd(CallbackInfo ci) {
-        int lastAnimTick = SpongeCommonEventFactory.lastAnimationPacketTick;
-        int lastPrimaryTick = SpongeCommonEventFactory.lastPrimaryPacketTick;
```

Comment:
The whole purpose of this code is to fire interact events while left-clicking AIR. If you remove this, there needs to be an alternative to handle these events or many plugins will break.